src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,850 @@
+{
```

Comment:
<ul><li>pyplot --&gt; matplotlib</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Corrected in a6b9f26f

Reply:
Is to be corrected when plotly becomes the goto librairy for the post_processing on those notebook

Reply:
Corrected in a492fccf